code/code/disc/disc_adv_adventuring.h

```diff
@@ -39,7 +40,8 @@ class CDAdvAdventuring : public CDiscipline
       skTrollish(),
       skBullywug(),
       skAvian(),
-      skKalysian()
+      skKalysian(),
+      skCommon()
```

Comment:
I suspect that these constructors don't do anything smart. Try removing, see if the autogenerated version works fine?

Reply:
I'll give it a go this weekend and provide feedback.